README.md

```diff
@@ -1,63 +1,64 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Recipe App
 
-# Project Name
-
-> Description the project.
+## Description
 
+> The Recipe app keeps track of all your recipes, ingredients, and inventory. It will allow you to save ingredients, keep track of what you have, create recipes, and generate a shopping list based on what you have and what you are missing from a recipe. Also, since sharing recipes is an important part of cooking the app should allow you to make them public so anyone can access them.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
+- Ruby
+- Ruby on Rails
+- Postgress
+- Capybara/Rspec
 
-## Getting Started
+## Live Demo (Link for Recipe App)
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+[Live Demo]()
```

Comment:
- [ ] If you will upload the live link kindly add it here, if not kindly delete it.